uber/site_sections/registration.py

```diff
@@ -132,8 +132,7 @@ def change_badge(self, session, id, message='', **params):
                 badge_num = int(params.get('newnum') or 0)
                 if c.AT_THE_CON and badge_num == 0:  # sometimes admins need to unset accidental badge assignments
                     attendee.badge_num = 0
-                else:
-                    message = session.change_badge(attendee, params['badge_type'], badge_num)
+                message = session.change_badge(attendee, params['badge_type'], badge_num)
```

Comment:
This broke because of a change we made earlier this week to fix a different bug :(

``session.change_badge`` used to be called all the time and we accidentally had it only get changed in the ``else`` block.  Whoops. Now it's called all the time.  I tested both setting and not setting a badge number to make sure it did the correct thing in both cases.